generators/client/templates/angular/src/main/webapp/content/scss/vendor.scss.ejs

```diff
@@ -23,9 +23,16 @@ put Sass variables here:
 eg $input-color: red;
 ****************************/
 // Override Boostrap variables
+<%_ if (clientTheme === 'none') { _%>
 @import "bootstrap-variables";
-// Import Bootstrap source files from node_modules
+<%_ } else {_%>
+@import "~bootswatch/dist/<%= clientTheme %>/variables";
```

Comment:
do you need this as you are also importing theme below that should include it as default?

Reply:
you're correct

Reply:
ok